src/book/01-team/05-robocup/03-debriefs.mdx

```diff
@@ -8,8 +8,168 @@ authors:
   - Ysobel Sims (@ysims)
   - Josephus Paye II (@JosephusPaye)
   - Lachlan Court (@LachlanCourt)
+  - Clayton Carlon (@Claegtun)
 ---
 
+## 2025: Salvador, Brazil
+
+### What went wrong
+
+#### Hardware
+
+- Necks broke often; screws would sometimes come out of the horn.
+- Frankie would brown out after falling.
+- Sarah's torso-pitch is off despite correct offsets.
+- Ethernet was inconsistent.
+- Head and neck often became loose.
+
+#### NUSense
+
+- USB connection would die and need power-cycle.
+- Baudrate had to be dropped from 3 Mbps to 1 Mbps.
+- Servos would sometimes move to the wrong positions.
+
+#### Architecture
+
+- Robot freezes with too many bugs. (Maybe NUClear bugs?)
```

Comment:
This is specifically network forwarding issues. NUClear issue with threads on network emits.

Reply:
Done.